gamestonk_terminal/common/technical_analysis/overlap_view.py

```diff
@@ -96,6 +89,13 @@ def view_ma(
     plt.show()
     print("")
 
+    export_data(
+        export,
+        os.path.dirname(os.path.abspath(__file__)),
+        f"{ma_type}{window_length}",
```

Comment:
Should we have ma_type.lower() ? And should we add stock name to make more sense? Also, how does the window_length look with multiple windows?

Apart from this, all good to me!